web/src/components/storage/DeviceSelector.test.jsx

```diff
@@ -0,0 +1,286 @@
+      // only once
+      await user.click(md0Option);
+      await user.click(md0Option);
+
+      expect(onSelectFn).toHaveBeenCalledTimes(1);
+      expect(onSelectFn).toHaveBeenCalledWith(md0);
+    });
+  });
+
+  describe("DeviceSelector options content", () => {
+    it("renders the device size", () => {
+      plainRender(<DeviceSelector devices={[vda]} />);
+      screen.getByText("1 KiB");
+    });
+
+    it("renders the device name", () => {
+      plainRender(<DeviceSelector devices={[vda]} />);
+      screen.getByText("/dev/vda");
+    });
+
+    it("renders the device model", () => {
+      plainRender(<DeviceSelector devices={[vda]} />);
+      screen.getByText("Micron 1100 SATA");
+    });
+
+    describe("when device is a SDCard", () => {
+      it("renders 'SD Card'", () => {
+        const sdCard = { ...vda, sdCard: true };
+        plainRender(<DeviceSelector devices={[sdCard]} />);
+        screen.getByText("SD Card");
+      });
+    });
+
+    describe("when content is given", () => {
+      it("renders the partition table info", () => {
+        plainRender(<DeviceSelector devices={[vda]} />);
+        screen.getByText("GPT with 2 partitions");
+      });
+
+      it("renders systems info", () => {
+        plainRender(<DeviceSelector devices={[vda]} />);
+        screen.getByText("Windows 11");
+        screen.getByText("openSUSE Leap 15.2");
+      });
+    });
+
+    describe("when content is not given", () => {
+      it("renders 'No content found'", () => {
+        plainRender(<DeviceSelector devices={[multipath]} />);
+        screen.getByText("No content found");
+      });
+    });
+
+    describe("when device is software RAID", () => {
+      it("renders its level", () => {
+        plainRender(<DeviceSelector devices={[md0]} />);
+        screen.getByText("Software RAID0");
+      });
+
+      it("renders its members", () => {
+        plainRender(<DeviceSelector devices={[md0]} />);
+        screen.getByText(/Members/);
+        screen.getByText(/vdb/);
+      });
+    });
+
+    describe("when device is RAID", () => {
+      it("renders its wires", () => {
+        plainRender(<DeviceSelector devices={[raid]} />);
+        screen.getByText(/Devices/);
+        screen.getByText(/sda/);
+        screen.getByText(/sdb/);
+      });
+    });
+
+    describe("when device is a multipath", () => {
+      it("renders 'Multipath'", () => {
+        plainRender(<DeviceSelector devices={[multipath]} />);
+        screen.getByText("Multipath");
+      });
+
+      it("renders its wires", () => {
+        plainRender(<DeviceSelector devices={[multipath]} />);
+        screen.getByText(/Wires/);
+        screen.getByText(/sdc/);
+        screen.getByText(/sdd/);
+      });
+    });
+
+    describe("when device is DASD", () => {
+      it("renders its bus id", () => {
+        plainRender(<DeviceSelector devices={[dasd]} />);
+        screen.getByText("DASD 0.0.0150");
+      });
+
+      it("renders its wires", () => {
```

Comment:
A DASD has no wires.

Reply:
A leftover from a copy&paster of multipath test. In fact, it renders the selector using the multipath device. Deleted.